### DISCUSSION TITLE
Lack of instructions for installing

### DISCUSSION THREAD

**Discussion by benwiggy** [SVG-Edit/svgedit#976] [General]
I could not find anything in the documentation that tells you how to get the software up and running. 

Loading index.html doesn't work in my Safari browser on macOS (blank white screen), so I've no idea if I should be doing anything else.

The 'online' version works in the same browser.

**Comment by jfhenon** [comment]
can you let us know what is in your console?

**Comment by benwiggy** [comment]
[Error] TypeError: Module name, '@svgedit/svgcanvas/common/browser' does not resolve to a valid URL.
[Error] Failed to load resource: The requested URL was not found on this server. (favicon.ico, line 0)

**Comment by jfhenon** [comment]
Did you follow instructions in the readme?
https://github.com/SVG-Edit/svgedit?tab=readme-ov-file#i-want-to-host-svgedit-in-my-environment

**Comment by benwiggy** [comment]
Apologies, that's a comprehension fail on my part.

 npm doesn't seem to be on macOS by default, so that's another thing to install. Running it remotely seems the easier option for now.

**Comment by brenoepics** [comment]
I made a Tauri App for SVG Edit https://github.com/brenoepics/svgedit-app

You could just download it from releases (automatically created with GitHub Actions) for Linux, Mac and Windows, I'd be down to open a PR and add it as a package in svgedit or moving it to svg-edit org if the maintainer wants so that on each release we also get an installable version which is just 6MB.
